plat/arm/board/common/board_arm_trusted_boot.c

```diff
@@ -7,7 +7,7 @@
 #include <arm_def.h>
 #include <assert.h>
 #include <platform.h>
-#include <platform_oid.h>
+#include <tbbr_oid.h>
```

Comment:
Please move the include to be in alphabetical order?
Also, please update the copyright header to 2015-2017.